<launch>

  <arg name="view_image" default="False" doc="Set to True to show the image from the camera on launch."/>

  <include file="$(find table_vision_sensing)/launch/start_ocam.launch">
    <arg name="view_image" value="$(arg view_image)"/>
  </include>

  <!-- <node name="background_removal" type="background_removal" pkg="table_vision_sensing" output="screen"/> -->

  <node name="player_tracking" type="player_tracking" pkg="table_vision_sensing" output="screen">
    <rosparam command="load" file="$(find table_vision_sensing)/config/player_angle_detection.yaml"/>
  </node>


</launch>
